database/migration/changesets/20200315_CreateWorkloadTable.xml

```diff
@@ -0,0 +1,123 @@
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="tbl@broadinstitute.org"
+               dbms="postgresql"
+               id="20200315"
+               logicalFilePath="20200315_CreateWorkloadTable.xml">
+        <comment>
+            Define a new root workload table schema.
+            There is a row in this table for every workload.
+            The 'table' column here names another table.
+            That other table has the composite Postgres type 'workflow',
+            and is named with the 'pipeline' column
+            composed with the 'id' column of this table.
+        </comment>
+        <rollback>
+            I have no idea.
+        </rollback>
+        <sql>
+            CREATE TYPE workflow AS ENUM ()
+            <comment>
+                Add a new value for each new 'table' type.
+                This new value should go
+                in the 'pipeline' column
+                for workloads of this kind.
+                See 20200316_ExternalWholeGenomeReprocessing.xml
+                for an example.
+            </comment>
+        </sql>
+        <createTable tableName="workload">
+            <column name="id" type="bigint" autoIncrement="true"
+                    remarks="Unique in every row.
+                             Compose this with the value
+                             in the 'pipeline' column
+                             to get the value
+                             in the 'table' column below.">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="commit" type="text"
+                    remarks="The Git commit hash of WorkFlowLauncher
+                             that managed this workload.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="timestamptz"
+                    defaultValueComputed="now()"
+                    remarks="When was the workload row created here?">
+                <constraints nullable="false"/>
+            </column>
+            <column name="creator" type="text"
+                    remarks="Who created this workload?">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cromwell" type="text"
+                    remarks="The URL of the Cromwell instance
+                             running the workflows.
+                             Should be a common prefix
+                             of the 'cromwell' column
+                             in the 'table' table.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="finished" type="timestamptz"
```

Comment:
What do you think about calling this `end_time`? It makes it clearer that this field refers to the time that the workflow finished and not whether it finished

Reply:
It is typed `timestamptz`, not `boolean`.
And we obviously cannot know when it finished until it's finished.